### ISSUE TITLE
Allow to configure when a repository is pruned

### ISSUE THREAD

**Issue by TribuneX** [ZettaIO/restic-compose-backup#38]
A prune operation can be a costly operation if backing up to a cloud location such as S3 or Backblaze B2. It would be beneficial if the prune operation could be also configured with a cron schedule in order to run prune operations less often than the actual backup.

I assume this is not possible yet?

**Comment by einarf** [comment]
Not possible right now as far as I can remember. You suggest making it configurable to prune every other day or once a weel for example?

**Comment by TribuneX** [comment]
Exactly. Since a prune operation can be costly if you need to pay for download operations. It can be worth pruning a repository only once in a while.
